src/content/reference/react/useContext.md

```diff
@@ -461,15 +461,15 @@ function LoginForm() {
   return (
     <>
       <label>
-        First name{': '}
+        Prénom{': '}
```

Comment:
```suggestion
        Prénom :{' '}
```

Trop chelou leur choix de forçage ici, au lieu du traditionnel `{' '}` pour juste l'espace de fin

Reply:
Oui, j'ai trouvé ça étrange aussi